.swift-format.config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "fileScopedDeclarationPrivacy": {
+    "accessLevel": "private"
+  },
+  "indentation": {
+    "spaces": 4
+  },
+  "indentConditionalCompilationBlocks": true,
+  "indentSwitchCaseLabels": false,
+  "lineBreakAroundMultilineExpressionChainComponents": false,
+  "lineBreakBeforeControlFlowKeywords": false,
+  "lineBreakBeforeEachArgument": true,
+  "lineBreakBeforeEachGenericRequirement": false,
+  "lineLength": 100,
```

Comment:
I liked the old 150 more, ore we could go with a shorter 130 but I wouldn't short too much

Reply:
This is the `swift-format`'s default but I can change it to 130